extensions/ql-vscode/src/stories/vscode-webview-ui-toolkit/Overview.stories.mdx

```diff
@@ -0,0 +1,9 @@
+<!-- Checkbox.stories.mdx -->
+
```

Comment:
Would be cool to have an Overview page like this one for the "MRVA" stories too.